src/Blog/View/posts.twig

```diff
@@ -15,8 +22,15 @@
 		<h1>Blog</h1>
 		<ul>
 			{% for post in posts %}
-				<li>
-					<a href="/blog/{{ post.id }}">{{ post.title }}</a>
+				<li class="u-margin-medium">
```

Comment:
Add "clearfix" class selector to this element because the childrens of this element are floated. It should look like this:
```
class="u-margin--medium clearfix"
```

Reply:
done